controller/test-files/iteration/update/ok_change_parent.golden.json

```diff
@@ -0,0 +1,52 @@
+{
+  "data": {
+    "attributes": {
+      "active_status": false,
+      "created-at": "0001-01-01T00:00:00Z",
+      "description": "Some description",
+      "name": "iteration 3",
+      "parent_path": "/d124787e-db62-4c95-9aca-ea0bfe32a6b1/67a614a0-e657-431b-a989-ec1f63fa43d3",
```

Comment:
Can you please change the test that produces this file to use the UUID agnostic compare function? Then future updates to this file won't show updates to any UUID, as long as the order in the file doesn't change.

Reply:
sure, the test was already using UUID agnostic compare, just needed to re-generate the golden file.
done in https://github.com/fabric8-services/fabric8-wit/pull/1770/commits/3ccaca03459814b927d7260a2a1896f96bf65ac5
Thanks.